lambdas/data-transfer/handler.py

```diff
@@ -43,6 +43,10 @@ def handler(event, context):
         filename = src_key.split("/")[-1]
 
         target_key = f"{object.get('collection')}/{filename}"
+        # The file-staging directory is the default for MAAP's bucket
```

Comment:
How was this working before, in order to have the file-staging folder actually show up in the `target_url`? 

My guess is it was contained in the `TARGET_BUCKET` string ?